### PR TITLE
#A17. As Admin, I would like to add a more complex description to the Hackathon View using HTML.

### PR DESCRIPTION
**Pull request by stefdworschak**
## Description
- Adding a new field `tag_line` which will be a short description shown on the list view
- Enabling HTML safe for the description field on the Hackathon detail view page
- Adding a dropdown option for `Mentor` in the `user_type` field
- Adding template tag for parsing a prettier Status for participant users

## Pull request type
- [ ] #M01 (Miscellaneous User Story #01)
- [ ] #P02 (Participant User Story #02)
- [ ] #S03 (Staff User Story #03)
- [x] #A04 (Admin User Story #04)
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Related Issue
#164

## Screenshot
image

### CODE REVIEW

File: hackathon/templates/hackathon/hackathon_view.html [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {% extends "base.html" %}
 {% load static %}
+{% load my_tags %}
```
Comment (TravelTimN) [inline hackathon/templates/hackathon/hackathon_view.html:3]: Acceptable, but too generic perhaps?
{% load custom_tags %}
Or some sort of tag-name reference to what they're meant for?
Willing to accept this as custom_tags, though.

File: hackathon/templatetags/my_tags.py [hunk 2]
```diff
@@ -69,3 +69,17 @@ def place_identifier(num):
...
+    if status == 'published':
+        return "Registration Starts Soon"
+    elif status == 'hack_in_progress':
+        return "Registration Open"
+    elif status == 'hack_in_progress':
+        return "Hackathon In Progress"
+    elif status == 'judging':
+        return "Judging In Progress"
+    else:
+        return "Hackathon Finished"
```
Comment (TravelTimN) [inline hackathon/templatetags/my_tags.py:85]: missing blank line at end of file

**Review by TravelTimN — APPROVED** [review]
LGTM - minor suggestions/change only.

File: hackathon/templates/hackathon/hackathon_view.html (hunk 1 above)
Comment (stefdworschak) [reply hackathon/templates/hackathon/hackathon_view.html]: Haha I did not create the file, but I agree. Renamed it to `custom_tags.py` and changed all the references to it.

File: hackathon/templatetags/my_tags.py (hunk 2 above)
Comment (stefdworschak) [reply hackathon/templatetags/my_tags.py]: Added the newline at the end of the file.

**Pull request merged**